.github/workflows/optimize-ci-build-reusable.yml

```
@@ -79,6 +79,13 @@ jobs:
         with:
           directory: ./optimize/client
 
+      #########################################################################
+      # Build optimize dependencies
+      - name: Build optimize dependencies
+        uses: ./.github/actions/run-maven
+        with:
+          parameters: -pl optimize/backend -am install -Dquickly -DskipTests -Dskip.docker -P skipFrontendBuild,runAssembly -T 1
```

```suggestion
          parameters: -pl optimize/backend -am install -Dquickly -P runAssembly -T 1
```
the quickly flag should also be enough here